File: C++/MathematicalPrograms/RuleOf72.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int principal, rate, doublerate, doubleprincipal;
	cout << "Enter the Principal Amount";
	cin >> principal;
	cout << "Enter the Rate of Interest";
	cin >> rate;
	doublerate = 72/rate;
	doubleprincipal = 2*principal;
	cout << "Your Principal Amount ₹ " << principal << " will become ₹ " << doubleprincipal << " in "  << doublerate << " years";
}

File: Java/MathematicalPrograms/RuleOf72.java
import java.util.Scanner;
class RuleOf72 {
    public static void main(String[] args) { {
        Scanner myObj = new Scanner(System.in);
        System.out.println("Enter the Principal Amount");
        double principal = myObj.nextInt();
        System.out.println("Enter the Rate of Interest");
        double rateofinterest = myObj.nextInt();
        System.out.println("Your Principal Amount ₹ " + principal + " will become ₹ " + 2*principal + " in " + 72/rateofinterest + " years");
    }
}
}